src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/51.diff.sql

```diff
@@ -0,0 +1,8 @@
+﻿-- Adding a new index so that the reindex job can make use of getting counts in large dbs
```

Comment:
Should we change this to create only if it does not exist?  If, for some reason a large customer had to re-run this bit of schema code, I would hate to drop and recreate this index once it has already been created.